.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         python-version: [3.6, 3.7, 3.8]
```

Comment:
Don't we need to drop 3.6 too?

Reply:
i guess thats ok.

You can also just comment out the github actions step for the linting. Not a big issue.

If you change the python and other deps could you reflect those changes in setup.py and requirements-package.txt?

Reply:
Regarding Python 3.6, I have tried to do it in a BC-way: if `contextvars` cannot be imported, the "chunked upload" feature is not available.

Of course I'm ok with removing compatibility with Python 3.6 altogether but it seemed a bit harsh.

Let me know what you decide.

Reply:
Commented out Black in Makefile "check" for now, as it is causing the lint issue.

Locally, black tells me that everything is fine...

Reply:
Can you comment out lines 62-64 in the github actions file instead? https://www.github.com/danielfrg/s3contents/tree/master/.github%2Fworkflows%2Ftest.yml

Reply:
Done